api/fixtures/example.go

```diff
@@ -408,6 +410,84 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 		}
 		services = getIngressServicePublishingStrategyMapping(o.NetworkType)
 
+	case o.PowerVS != nil:
+		buildIBMCloudCreds := func(name, apikey string) *corev1.Secret {
+			return &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: namespace.Name,
+					Name:      name,
+				},
+				Data: map[string][]byte{
+					"ibm-credentials.env": []byte(fmt.Sprintf(`IBMCLOUD_AUTH_TYPE=iam
+ IBMCLOUD_APIKEY=%s
+ IBMCLOUD_AUTH_URL=https://iam.cloud.ibm.com
+ `, apikey)),
+				},
+			}
+		}
+		powerVSResources := &ExamplePowerVSResources{
+			buildIBMCloudCreds(o.Name+"-cloud-ctrl-creds", o.PowerVS.ApiKey),
+			buildIBMCloudCreds(o.Name+"-node-mgmt-creds", o.PowerVS.ApiKey),
+			buildIBMCloudCreds(o.Name+"-cpo-creds", o.PowerVS.ApiKey),
+		}
+		resources = powerVSResources.AsObjects()
+		platformSpec = hyperv1.PlatformSpec{
+			Type: hyperv1.PowerVSPlatform,
+			PowerVS: &hyperv1.PowerVSPlatformSpec{
+				ResourceGroup:     o.PowerVS.ResourceGroup,
+				Region:            o.PowerVS.Region,
+				Zone:              o.PowerVS.Zone,
+				ServiceInstanceID: o.PowerVS.CloudInstanceID,
+				Subnet: &hyperv1.PowerVSResourceReference{
+					Name: &o.PowerVS.Subnet,
+					ID:   &o.PowerVS.SubnetID,
+				},
+				VPC: &hyperv1.PowerVSVPC{
+					Name:   o.PowerVS.Vpc,
+					Region: o.PowerVS.VpcRegion,
+					Subnet: o.PowerVS.VpcSubnet,
+				},
+				KubeCloudControllerCreds:  corev1.LocalObjectReference{Name: powerVSResources.KubeCloudControllerPowerVSCreds.Name},
+				NodePoolManagementCreds:   corev1.LocalObjectReference{Name: powerVSResources.NodePoolManagementPowerVSCreds.Name},
+				ControlPlaneOperatorCreds: corev1.LocalObjectReference{Name: powerVSResources.ControlPlaneOperatorPowerVSCreds.Name},
+			},
+		}
+		services = []hyperv1.ServicePublishingStrategyMapping{
```

Comment:
any reason to not use `getIngressServicePublishingStrategyMapping(o.NetworkType)`?

Reply:
We can use `getIngressServicePublishingStrategyMapping()` func itself. Modified the code to use this func.